### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/nagioscore/actions/runs/4776228849#summary-12951153328

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/nagioscore/actions/runs/4776228989#summary-12951153517

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ This document serves as our guidelines for contribution.
 
 ## Questions
 
-If you have a question, your best option is to ask on the [Nagios Community Support Forums](https://support.nagios.com/forum/viewforum.php?f=7). If you leave a question in the Issues list on Github, you will likely be redirected to the forums.
+If you have a question, your best option is to ask on the [Nagios Community Support Forums](https://support.nagios.com/forum/viewforum.php?f=7). If you leave a question in the Issues list on GitHub, you will likely be redirected to the forums.
```
Comment (jsoref) [inline CONTRIBUTING.md:8]: brand

File: Makefile.in [hunk 2]
```diff
@@ -493,4 +493,4 @@ dist: distclean
 	rm -f nagios-$(VERSION)
 
 # Targets that always get built
-.PHONY:	indent clean clean distclean dox test html
+.PHONY:	indent clean distclean dox test html
```
Comment (jsoref) [inline Makefile.in:496]: sometimes duplicates can be intentional (there's some autoconf stuff that has 2 or 5 instances of `false` in a row...). My (admittedly rusty) recollection of make doesn't indicate this to be one of those times...

File: base/config.c [hunk 3]
```diff
@@ -628,7 +628,7 @@ int read_main_config_file(char *main_config_file) {
 				}
 			}
 
-		else if(!strcmp(variable, "use_agressive_host_checking") || !strcmp(variable, "use_aggressive_host_checking")) {
+		else if(!strcmp(variable, "use_aggressive_host_checking") || !strcmp(variable, "use_aggressive_host_checking")) {
```
Comment (jsoref) [inline base/config.c:631]: Sigh, this is almost certainly a backwards compat thing. I'll drop.

File: base/config.c [hunk 4]
```diff
@@ -1755,7 +1755,7 @@ int pre_flight_object_check(int *w, int *e) {
 			temp_host->notification_period_ptr = temp_timeperiod;
 			}
 
-		/* check all parent parent host */
+		/* check all parents of host */
```
Comment (jsoref) [inline base/config.c:1758]: I spent some time thinking about this and this is my best guess, but I could very easily be wrong

File: cgi/config.c [hunk 5]
```diff
@@ -1779,8 +1779,8 @@ void display_servicedependencies(void) {
 		return;
 		}
 
-	printf("<P><DIV ALIGN=CENTER CLASS='dataTitle'>Service Dependencie%s%s</DIV></P>\n",
-	       (*to_expand == '\0' ? "s" : "s Involving Host "), (*to_expand == '\0' ? "" : html_encode(to_expand, FALSE)));
+	printf("<P><DIV ALIGN=CENTER CLASS='dataTitle'>Service Dependencies%s%s</DIV></P>\n",
+	       (*to_expand == '\0' ? "" : " Involving Host "), (*to_expand == '\0' ? "" : html_encode(to_expand, FALSE)));
```
Comment (jsoref) [inline cgi/config.c:1783]: I don't see the value of intentionally using an incomplete word here. Especially when the second half of the substitution was quite willing to use `""`.

File: cgi/jsonutils.c [hunk 6]
```diff
@@ -602,7 +602,7 @@ void json_array_append_duration(json_object *obj, unsigned long value) {
 	list of nodes. Nodes may be either a key or a zero-based array index.
 
 	For example to return the query_time key in the result object, the
-	path would be "result.query_time". To find the 2nd host host in
+	path would be "result.query_time". To find the 2nd host in
```
Comment (jsoref) [inline cgi/jsonutils.c:605]: ??

File: common/objects.c [hunk 7]
```diff
@@ -537,7 +537,7 @@ daterange *add_exception_to_timeperiod(timeperiod *period, int type, int syear,
 	if(period == NULL)
 		return NULL;
 
-	/* allocate memory for the date range range */
+	/* allocate memory for the date range */
```
Comment (jsoref) [inline common/objects.c:540]: ??

It's possible that `daterange` is intended, and perhaps `daterange range`, but neither seemed great...

File: include/archiveutils.h [hunk 8]
```diff
@@ -114,7 +114,7 @@
 									AU_NOTIFICATION_SERVICE_FLAPPING_STOP | \
 									AU_NOTIFICATION_SERVICE_UNKNOWN)
 
-#define AU_NOTFICATION_ALL	(AU_NOTFICATION_HOST_ALL | \
+#define AU_NOTIFICATION_ALL	(AU_NOTIFICATION_HOST_ALL | \
 							AU_NOTIFICATION_SERVICE_ALL)
```
Comment (jsoref) [inline include/archiveutils.h:118]: ??

File: lib/fanout.h [hunk 9]
```diff
@@ -43,7 +43,7 @@ extern void fanout_destroy(fanout_table *t, void (*destructor)(void *));
  * Return a pointer from the fanout table t
  *
  * @param[in] t table to fetch from
- * @param[in] key key to fetch
+ * @param[in] key Key to fetch
```
Comment (jsoref) [inline lib/fanout.h:46]: This matches the style used for `fanout_add` below

File: t-tap/test_checks.c [hunk 10]
```diff
@@ -1278,7 +1278,7 @@ void run_passive_host_tests()
     ok(hst1->last_hard_state == STATE_DOWN,
         "Host last hard state was down");
     ok(hst1->no_more_notifications == TRUE,
-        "No more notifications reatained");
+        "No more notifications retained");
```
Comment (jsoref) [inline t-tap/test_checks.c:1281]: ??

File: xdata/xodtemplate.h [hunk 11]
```diff
@@ -828,7 +828,6 @@ int xodtemplate_resolve_hostescalation(xodtemplate_hostescalation *);
 int xodtemplate_resolve_hostextinfo(xodtemplate_hostextinfo *);
 int xodtemplate_resolve_serviceextinfo(xodtemplate_serviceextinfo *);
 
-int xodtemplate_merge_extinfo_ojects(void);
```
Comment (jsoref) [inline xdata/xodtemplate.h:831]: My default correction would be `objects`, but the item appears unused, so instead I've just removed it...

**Review by jsoref — COMMENTED** [review]
Most corrections automatically suggested by Google Sheets. All fault mine

**Comment by sawolf** [comment]
Hi Josh, thanks for the fixes! Merging shortly

**Pull request merged**